Core/Models/PointMassSecondOrder.hpp

```diff
@@ -67,13 +74,23 @@ namespace gtfo{
             const Scalar &dt = parameters.dt;
             const Scalar &mass = parameters.mass;
             const Scalar &damping = parameters.damping;
+            const Scalar &stiffness = parameters.stiffness;
```

Comment:
Please also modify the old code to have `const Scalar&` instead of `const Scalar &` for consistency with the rest of the library